gdal/frmts/gdalallregister.cpp

```diff
@@ -526,6 +526,10 @@ void CPL_STDCALL GDALAllRegister()
     GDALRegister_JPEGLS();
 #endif
 
+#ifdef FRMT_jpegxr
```

Comment:
Please explain why you are putting it here in the ordering.  No need for a comment, but it is good to have in a review comment.